ThatConference/that-website
# Use UTC dates when rendering the Key Dates section

Fixes #162 

Yay timezones.

Testing:

1. Load call for Counselors http://localhost:3000/wi/call-for-counselors?feature=callForCounselors
2. Make sure site shows 1/1/2020 for the Call for Counselors submission.
